src/libcrun/container.c

```diff
@@ -2084,6 +2084,19 @@ libcrun_container_exec (libcrun_context_t *context, const char *id, oci_containe
             libcrun_warning ("cannot set HOME environment variable");
         }
 
+      /* If the new process block doesn't specify a SELinux label or AppArmor profile, then
+         use the configuration from the original config file.  */
+      if (container->container_def->process)
+        {
+          if (process->selinux_label == NULL && container->container_def->process->selinux_label)
+            process->selinux_label = container->container_def->process->selinux_label;
+        }
+      if (container->container_def->process)
```

Comment:
a bit late now, but why the separate block here?

Reply:
copy&pasta :-)

Reply:
could you check the second patch in https://github.com/containers/crun/pull/214 ?